Angular support

vltansky opened the pull request:
#3

vltansky commented:
Take a look when you have time.
`npm run build:angular`, `npm run demo:angular`.

nolimits4web commented on package.json:
```diff
@@ -39,55 +41,87 @@
+    "rxjs": "~6.6.0",
+    "tslib": "^2.0.0",
+    "zone.js": "~0.10.2"
+  },
```
Do we need all of this in dependencies and not in devDependencies?

vltansky replied:
No, I just thought it's an internal package.json and it's not really matters. It can be moved to dev deps.

nolimits4web commented:
Hi, thanks for your PR! Seems to be it is working fine (at least the demo 😄  ). Have few notes/questions:
- Do we really need all the test (karma) related files and dependencies, as i see we don't really need tests here (or at least the current ones don't test much)
- Under git control in `package/angular` there shouldn't be anything else except `package.json` and `README.md`
- I see it currently builds the UMD bundle as a package `main` entry, but i believe we don't need UMD and UMD is not "tree-shakeable". Can we build here CJS format like it is done for React/Vue/Svelte and how you done with ESM builds for Angular?

vltansky commented:
1. Removed
2. Updated the `.gitignore`, is `README.md` should be copied there?
3. ng-packagr (library bundler) follows the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/mobilebasic). related: https://github.com/ng-packagr/ng-packagr/issues/460.
4. I think we need to add `tslib` as a dependency. According to https://angular.io/guide/migration-update-libraries-tslib

> The tslib is a runtime library for Typescript. The version of this library is bound to the version of the TypeScript compiler used to compile a library. Peer dependencies do not accurately represent this relationship between the runtime and the compiler. If tslib remained declared as a library peer dependency, it would be possible for some Angular workspaces to get into a state where the workspace could not satisfy tslib peer dependency requirements for multiple libraries, resulting in build-time or run-time errors.
> 
> As of TypeScript 3.9 (used by Angular v10), tslib version of 2.x is required to build new applications. However, older libraries built with previous version of TypeScript and already published to npm might need tslib 1.x. This migration makes it possible for code depending on incompatible versions of the tslib runtime library to remain interoperable.

nolimits4web commented:
Merged, thanks. Yes, those readmes are leftovers from previous version.

nolimits4web commented:
Can you also help me with adding docs for Angular to https://github.com/nolimits4web/skeleton-elements-website ?

vltansky commented:
Yeah of course, will make a PR soon
